tools/shared/src/main/scala/org/scalajs/core/tools/linker/LinkingUnit.scala

```diff
@@ -11,8 +11,8 @@ final class LinkingUnit private[linker] (
     val esLevel: ESLevel,
     val classDefs: List[LinkedClass],
     private[linker] val infosInternal: Map[String, Infos.ClassInfo],
-    val isComplete: Boolean,
-    dummyInternal: Unit
+    val entryPoints: List[EntryPoint],
```

Comment:
Currently you get a linking error saying:
```
[error] Referring to non-existent class org.scalajs.testsuite.compiler.EntryPointInNoConfigurationFoo$
[error]   called from core module entry points
[error] Referring to non-existent method org.scalajs.testsuite.compiler.EntryPointInNoConfigurationFoo$.main()scala.Unit
[error]   called from core module entry points
```
IMO this is good enough, don't you think?